doc/intro.rst

```diff
@@ -130,10 +130,25 @@ in turn start up the device server process, all in one go.
     $ tango-simlib-launcher --name mkat_sim/weather/1 --class Weather\
                             --name mkat_simcontrol/weather/1\
                             --class WeatherSimControl\
-                            --server-command weather-DS --port 0\
+                            --server-command ./weather-DS --port 0\
```

Comment:
I forgot to ask this: passing `weather-DS` instead of `./weather-DS` should still work. So why the change?

Reply:
When I was testing out this command, it failed because `weather-DS` wasn't on the path.  Adding the `./` just helps the example work a little more easily.

Reply:
okay. thanks